### PR TITLE
Fixing imports

### PR DESCRIPTION
**Pull request by AbdelrahmanAbdelgwad**
###### Ticket(s): [#XXX](https://app.clickup.com/XXX)

- 🔄 **Main changes in this PR**
    - Implemented the soft actions in the `CarRacing` environment in the gym module
    - Documented how to install the package successfully
    - Solved the virtual env issue and documented the solution in the README file

- ⚠️ **Breaking changes**
  - Added the gym package, modified by `NotAnyMike` , as a module inside the haptic package and changed all import names to begin with `haptic.`
  - Adopted a src-layout for packaging

## 🚀 How to test this Pull Request

1. No tests are made yet.

## 📝 Notes for future improvements

- Write a more professional README file
- Find a way to install gym package alongside haptic package instead of just adding the gym package to the haptic package as stated above

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,47 @@
 ![tests](https://github.com/AbdelrahmanAbdelgwad/haptic/workflows/test/badge.svg)
 # Haptic
-Haptic feedback controll to assistant driving systems
+Haptic Shared Control to assist driving
```
Comment (youssiefanas) [inline README.md:3]: Shared autonomy using haptic feedback as a driver assistance system

File: src/haptic/__main__.py [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+        pass
+
+    def _on_training_end(self) -> None:
+        """
+        This event is triggered before exiting the `learn()` method.
+        """
+        pass
+
+
+def add(x, y):
```
Comment (youssiefanas) [inline src/haptic/__main__.py:127]: ما تنضف الكلام ده

**Review by youssiefanas — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (AbdelrahmanAbdelgwad) [reply README.md:3]: Done

File: src/haptic/__main__.py (hunk 2 above)
Comment (AbdelrahmanAbdelgwad) [reply src/haptic/__main__.py:127]: This was for the sake of only testing the CI until we write actual tests for our code

**Pull request merged**